desktop/src/app/components/gallery/home/gallery.html

```diff
@@ -2,36 +2,13 @@
 <bl-local-template-drop-zone *ngIf="autoStorageService.hasArmAutoStorage | async">
     <div class="header">
         <h1>{{'gallery.title' | i18n }}</h1>
-        <bl-refresh-btn [refresh]="refresh" type="round"></bl-refresh-btn>
-        <div class="flex-separator"></div>
-        <bl-local-template-pick-button></bl-local-template-pick-button>
     </div>
     <div class="gallery-content">
-        <div class="gallery-applications">
-            <div class="filter">
-                <input autofocus blInput [formControl]="quicksearch" placeholder="Search">
+        <bl-banner class="gallery-deprecation-warning" type="warning">
+            <div message>
+                Graphics and rendering pools have been retired on 29 February 2024.
+                <a class="gallery-warning-alternative-link" (click)="openLink('https://azure.microsoft.com/en-us/updates/azure-batch-rendering-vm-images-licensing-will-be-retired-on-29-february-2024/')" href="javascript:void(0)">Click for more info</a>
```

Comment:
Remove "en-us" from the URL so it can be localized:

https://azure.microsoft.com/updates/azure-batch-rendering-vm-images-licensing-will-be-retired-on-29-february-2024/

Reply:
Updated